src/ServiceManagement/Sql/Commands.SqlDatabase/Database/Cmdlet/StartAzureSqlDatabaseExport.cs

```diff
@@ -66,7 +64,7 @@ public class StartAzureSqlDatabaseExport : SqlDatabaseCmdletBase
             ParameterSetName = ByContainerObjectParameterSet,
             HelpMessage = "The Azure Storage Container to place the blob in.")]
         [ValidateNotNull]
-        public AzureStorageContainer StorageContainer { get; set; }
+        public CloudBlobContainer StorageContainer { get; set; }
```

Comment:
This will be a breaking change - I'm concerned about the differences between creating an AzureStorageConatiner and a CloudBlobContainer.  Does this break any piping scenarios?  What does it look like for a customer to create this parameter?